bitrise/print_test.go

```diff
@@ -100,7 +100,7 @@ func TestGetTrimmedStepName(t *testing.T) {
 		}
 
 		actual := getTrimmedStepName(result)
-		expected := ""
+		expected := "(0)                          (…, this is a very long string.)"
```

Comment:
remove every test where stepInfoModel.Title is empty (it is always filled)

Reply:
long step id + long id@version composite = max half space step title + max half space composite

+test